Modules/EMCAL/include/EMCAL/PedestalChannelCheck.h

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   PedestalChannelCheck.h
+/// \author Sierra Cantway
+///
+
+#ifndef QC_MODULE_EMCAL_EMCALPEDESTALCHANNELCHECK_H
+#define QC_MODULE_EMCAL_EMCALPEDESTALCHANNELCHECK_H
+
+#include "QualityControl/CheckInterface.h"
+
+namespace o2::quality_control_modules::emcal
+{
+
+/// \brief  Check whether a plot is good or not.
+///
+/// \author Sierra Cantway
```

Comment:
Doxygen comments should start with

```c++
/// \class PedestalChannelCheck
/// \brief Check for large amount of pedestal outliers in pedestal histograms 
/// \ingroup EMCALQCCheckers
/// \author Sierra Cantway
/// \since May 22nd, 2024
///
/// A bit more detailed description here ...
```

Also the comment `Check whether a plot is good or not.` is not really useful.

Obviously the parameters should be adapted for each case. Checkers are in the group `EMCALQCCheckers`, which is defined in `doxymodules.h`.